Formula/phoronix-test-suite.rb

```diff
@@ -29,38 +30,38 @@ def install
  mkdir -p $DESTDIR$INSTALL_PREFIX/share/doc/phoronix-test-suite/
 -mkdir -p $DESTDIR$INSTALL_PREFIX/../etc/bash_completion.d/
 +mkdir -p $DESTDIR$INSTALL_PREFIX/etc/bash_completion.d/
- 
```

Comment:
Why is the patch still necessary? Has it been reported upstream? Has it been accepted/merged?